spreadsheet_oca/static/src/spreadsheet/bundle/odoo_panels.esm.js

```diff
@@ -0,0 +1,98 @@
+/** @odoo-module */
+
+import * as spreadsheet from "@odoo/o-spreadsheet";
+import {Domain} from "@web/core/domain";
+import {Many2XAutocomplete} from "@web/views/fields/relational_utils";
+import {useService} from "@web/core/utils/hooks";
+
+const {chartSidePanelComponentRegistry} = spreadsheet.registries;
+const {LineBarPieDesignPanel} = spreadsheet.components;
+const {Component} = owl;
+
+export class OdooPanel extends Component {
+  setup() {
+    this.menus = useService("menu");
+  }
+  get menuProps() {
+    const menu = this.env.model.getters.getChartOdooMenu(this.props.figureId);
+    var result = {
+      record: this.record,
```

Comment:
Causes crash when adding a new chart to sheet.
Replace with `fieldString: _t("Menu Items"),`